src/pages/Profile.js

```diff
@@ -76,14 +76,22 @@ export default class Profile extends React.Component {
         }
       )
       .then(
-        () => this.props.portalStore.fetchProfileCharacters(this.state.id).then(
+        () => this.props.portalStore.fetchProfileCharacters(this.state.id,0).then(
           (characters) => {
             this.setState({
               characters: characters,
             })
           }
         )
       )
+      .then(
+        () => this.props.portalStore.fetchProfileCharacters(this.state.id,1).then(
```

Comment:
and here ;)

Reply:
Should `dead` argument be `1` or `true`? 

[Here](https://github.com/dnd-al-krk/portal_frontend/pull/132/files#diff-4580e5222aea94ff83b6b41745c9ac02R118) we have `false`